### PR TITLE
clarify the definition of a Vector3

### PR DESCRIPTION
**Pull request by vrabaud**
This was asked in https://github.com/ros/geometry_experimental/pull/106

### CODE REVIEW

File: geometry_msgs/msg/Vector3.msg [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # This represents a vector in free space. 
+# It is only meant to represent a direction. Therefore, it does not
+# make sense to apply a translation to it (e.g., when applying a 
+# generic rigid transformation to a Vecto3, tf2 will only apply the
```
Comment (tfoote) [inline geometry_msgs/msg/Vector3.msg]: Typo Vecto(r)3

**Comment by vrabaud** [comment]
fixed typo. Should I cherry-pick to indigo and hydro ?

**Comment by tfoote** [comment]
Yeah, cherry-picking would be good. It's been true all along.

**Comment by vrabaud** [comment]
ok, cherry-picked with my super powers.

**Comment by tfoote** [comment]
Great thanks!

**Pull request merged**